### ISSUE TITLE
RE2 enemy skins issues 

### ISSUE THREAD

**Issue by tyrant20202** [IntelOrca/biorand#547]
### Version

BioRand v3.1.1

### Seed

all seeds

### Scenario

RE 1 (Chris)

### Player Character

all characters

### What happened?

A lot of enemy skins are not randomizing no matter the seed i use.
jill(RE3),Kendo, Kendo(RE2R) Kevin, Krauser, Leon(RE2R) Leon(RE4) Lisa Trevor(RE1R) Marvin, Marvin(RE2R) Mikhail, MrX, Murphy, Nemesis, Nemesis(2) Nemesis(RE3) Piers, Rebecca, Regina, Richard, sherry(RE2R) Sheva, Steve, Tyrant(CV) Tyrant(RE1) Tyrell, Wesker, Wesker(CV) Wesker(RE1R)    Wesker(RE5) William, Worm(RE3) Yawn(RE1) Zombies(RE3)

**Comment by IntelOrca** [comment]
This is unfortunately not enough information for me to investigate. Have you tried a seed with just one enemy skin selected such as Nemesis (RE 3)?

**Comment by tyrant20202** [comment]
I tried that. all the skins i named did not changed any letters or numbers in any seed I tried.  this for an example. R720-WW5F-WXPE806YWU431BNVH3021600C000000000H0Y00000. I see when we are making a seed, whatever we choose to randomize makes changes to those letters or numbers. the skins i listed are the only ones not working for me. I've checked and unchecked every thing in the rando and I see the change in the seed.  But the skins I listed  jill(RE3) on down did not change anything. Alexia is the only enemy skin that has changed tyrant 1 (MRX)
